src/kolibri-cli.ts

```diff
@@ -106,10 +106,19 @@ export class KolibriCli extends Program {
         }
     }
 
+    resultWrapper(result: any) {
+        try {
+          return process.env.KOLIBRI_CLI_FORCE_RAW === '1' ? JSON.stringify(result) : result;
+        }
+        catch(_e){
+          return result;
+        }
+    }
+
     async login(args: LoginParams) {
         const result = await this.client?.login(args);
         this.loggedIn = true;
-        return result;
+        return this.resultWrapper(result);
```

Comment:
Instead of wrapping the result in every method, you could do the check for the variable in the "then" method on line 71.
Something like:

         .then(async (result: unknown) => {
                if (result !== undefined) {
                    console.log(raw ? JSON.stringify(result) : result);
                }
                if (!this.isRepl()) {
                    await this.client?.disconnect();
                    process.exit();
                }
            })

Reply:
Done